src/components/editInput/editInput.scss

```diff
@@ -7,23 +7,18 @@
         width: 100%;
         color: #afb9c3;
         padding: 0px 8px;
+        outline: none;
     }
 
-    .radio:not(:last-child) label {
-        margin-bottom: 16px;
-    }
+    input[type="radio"] { outline: none; }
```

Comment:
Instead of placing this override here. Add a line in the index.scss file to handle this issue across the application.

```
input,
textarea { outline: none }
```